### ISSUE TITLE
CLI restore does not get password by default

### ISSUE THREAD

**Issue by aghamir** [pooler/electrum-ltc#198]
Hi,
I used `restore` cli to restore my private key--which was created by electrum-ltc-- to electrum-ltc. However, it doesn't get password to encrypt file by default. Moreover, when I use `--password` argument, I get this error:
```
Traceback (most recent call last):
  File "/usr/local/bin/electrum-ltc", line 392, in <module>
    result = run_offline_command(config, config_options, plugins)
  File "/usr/local/bin/electrum-ltc", line 241, in run_offline_command
    result = func(*args, **kwargs)
  File "/usr/local/lib/python3.6/dist-packages/electrum_ltc/commands.py", line 97, in func_wrapper
    return func(*args, **kwargs)
  File "/usr/local/lib/python3.6/dist-packages/electrum_ltc/commands.py", line 194, in restore
    wallet.update_password(old_pw=None, new_pw=password, encrypt_storage=encrypt_file)
  File "/usr/local/lib/python3.6/dist-packages/electrum_ltc/wallet.py", line 1109, in update_password
    self._update_password_for_keystore(old_pw, new_pw)
  File "/usr/local/lib/python3.6/dist-packages/electrum_ltc/wallet.py", line 1203, in _update_password_for_keystore
    self.keystore.update_password(old_pw, new_pw)
  File "/usr/local/lib/python3.6/dist-packages/electrum_ltc/keystore.py", line 188, in update_password
    self.check_password(old_password)
  File "/usr/local/lib/python3.6/dist-packages/electrum_ltc/keystore.py", line 152, in check_password
    self.get_private_key(pubkey, password)
  File "/usr/local/lib/python3.6/dist-packages/electrum_ltc/keystore.py", line 172, in get_private_key
    txin_type, privkey, compressed = deserialize_privkey(sec)
  File "/usr/local/lib/python3.6/dist-packages/electrum_ltc/bitcoin.py", line 468, in deserialize_privkey
    .format(neutered_privkey))
electrum_ltc.util.BitcoinException: cannot deserialize privkey H53..g=
```

**Comment by pooler** [comment]
I just tried restoring a private key using Electrum (for Bitcoin) and I obtained the same result, so this does not seem to be specific to Electrum-LTC. Please report upstream.

**Comment by aghamir** [comment]
Thanks a lot,
I worked with electrum 3.2.3 and there wasn't this issue. I checked the master in upstream, and as you told there is the problem.
BTW, can I use the electrum(3.2.3) to create wallet for electrum-ltc?

**Comment by pooler** [comment]
Fixed in 621866d.
